pom.xml

```diff
@@ -1289,11 +1290,25 @@
             <dependency>
                 <groupId>org.testcontainers</groupId>
                 <artifactId>testcontainers-bom</artifactId>
-                <version>1.14.3</version>
+                <!-- TODO update to released version. The update to RC was urgently needed due to https://github.com/testcontainers/testcontainers-java/issues/3166 -->
+                <version>${dep.testcontainers.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
+                <groupId>org.testcontainers</groupId>
+                <artifactId>testcontainers</artifactId>
+                <version>${dep.testcontainers.version}</version>
+            </dependency>
+
+            <dependency>
+                <!-- org.testcontainers:testcontainer's dependencies pull two different versions of this artifact and this is to negotiate the version -->
```

Comment:
Is this a known issue for testcontainers?

Reply:
this i do not know

Reply:
i mentioned this to @bsideup over slack

Reply:
FTR reporting an issue (so that it can be tracked and even picked up by the community) is always preferred over mentioning things on Slack :)